frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -114,6 +114,21 @@ export const getSurveyMetricsQueries = (surveyId: string): SurveyMetricsQueries
     }
 }
 
+export const getSurveyDataVizQuery = (survey: Survey): InsightVizNode => {
+    return {
+        kind: NodeKind.InsightVizNode,
+        source: {
+            kind: NodeKind.TrendsQuery,
+            // dateRange: { date_from: dayjs(survey.created_at).format('YYYY-MM-DD') },
+            //  properties: [{ type: PropertyFilterType.Event, key: '$survey_id', operator: PropertyOperator.Exact, value: survey.id }],
```

Comment:
There is a line in the schema types file that says `TrendsQuery` insight viz nodes are not yet implemented, just wanted to check if that's still true 👀 because some of it works and some of it doesn't

But basically here I just want to filter for the event property `survey_id` and allow the user to see some sick product analytics visuals 😁

Reply:
I think you're referring to the following comment? If so, that's just a description how the `TrendsFilterType` is build by subtracting the set of properties also existing in `FilterType`. All insights should work :)

```ts
/** `TrendsFilterType` minus everything inherited from `FilterType` and
 * `hidden_legend_keys` replaced by `hidden_legend_indexes` */
```

Reply:
<img width="436" alt="Screen Shot 2023-07-24 at 10 26 02 PM" src="https://github.com/PostHog/posthog/assets/25164963/6201cce5-cc9b-4ae4-ad4d-e4f21eb0960d">